publish.sh

```diff
@@ -1,19 +1,13 @@
 #!/usr/bin/env sh
 
+# normal package for react native
```

Comment:
Easier to publish UI package first rather than publish web and transform package.json back to UI version